### PR TITLE
Tickets/DM-8295

### PR DESCRIPTION
**Pull request by stargaser**
Setting up ws4py as a third-party package

### CODE REVIEW

File: ups/ws4py.table [hunk 1]
```diff
@@ -0,0 +1,6 @@
+setupRequired(python)
+
+envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(LSST_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```
Comment (timj) [inline ups/ws4py.table]: I don't think you need these `/lib` paths to be set.

**Review by timj — APPROVED** [review]
Please squash the commits. Also, add a note to the commit message if this really is an unreleased version (as indicated by the SHA1 in the tar file name).

File: ups/ws4py.table (hunk 1 above)
Comment (stargaser) [reply ups/ws4py.table]: I think you are right...I was copying the template without thinking.

**Comment by stargaser** [comment]
The weird part about the 0.3.5 version is -- it has not been tagged as a release in the git repo, but it was released as 0.3.5 on PyPI. https://github.com/Lawouach/WebSocket-for-Python/issues/192 indicates the lack of a tag is an oversight by the author.

**Pull request merged**